Packs/CommonScripts/Scripts/script-emailFieldTriggered.yml

```diff
@@ -14,11 +14,14 @@ script: |-
   if (email) {
       var body = 'Hello,\nField ' + args.name + ' was changed from ' + args.old + ' to ' + args.new + ' in incident ' + incidents[0].name + '.\n--DBot';
       var subject = 'Field ' + args.name + ' was triggered';
-      executeCommand('SendEmail', {'to':email, 'subject':subject, 'body':body});
+      executeCommand('send-mail', {'to':email, 'subject':subject, 'body':body});
```

Comment:
did you test that it works?

Reply:
@yuvalbenshalom seems to work. A lot of scripts are executing `send-mail` command, so don't think it should be a problem.